src/pages/messages/components/TicketListItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { RegistrationListItem } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { Ticket, PublishingTicket } from '../../../types/publication_types/messages.types';
+import { Registration, emptyRegistration } from '../../../types/registration.types';
+import { PublishingRequestMessagesColumn } from './PublishingRequestMessagesColumn';
+
+const ticketColor = {
+  PublishingRequest: 'publishingRequest.main',
+  DoiRequest: 'doiRequest.main',
+  GeneralSupportCase: 'generalSupportCase.main',
+};
+
+interface TicketListItemProps {
+  ticket: Ticket;
+}
+
+export const TicketListItem = ({ ticket }: TicketListItemProps) => {
+  const { t } = useTranslation();
+
+  const { id, identifier, mainTitle, contributors, publicationInstance } = ticket.publication;
+  const registration: Registration = {
+    ...emptyRegistration,
+    identifier,
+    id,
+    entityDescription: emptyRegistration.entityDescription
+      ? {
+          ...emptyRegistration.entityDescription,
+          mainTitle,
+          contributors,
+        }
+      : undefined,
+  };
+  if (registration.entityDescription?.reference?.publicationInstance) {
+    registration.entityDescription.reference.publicationInstance.type = publicationInstance.type;
+  }
```

Comment:
Bruker den minimalistiske publication vi får på ticket fra søket til å lage en `Registration`, slik at vi kan bruke samme komponent som for resultat-søk. Da vet vi at det ser likt ut begge plasser

Reply:
NB! det er en liten bug som må fikses her. Hvis søket ved en feil ikke returnerer instance type, så vil denne sette en type basert på noe annet 😬 må nøste opp i det

Reply:
Problemet oppstår når man ber om support FØR man velger type. Da er det jo obv ikke noen type der...